### ISSUE TITLE
Object#timeout is deprecated, use Timeout.timeout instead

### ISSUE THREAD

**Issue by runningman84** [sensu-plugins/sensu-plugins-network-checks#32]
```
/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-network-checks-0.2.4/bin/check-banner.rb:128:in `acquire_no_banner': Object#timeout is deprecated, use Timeout.timeout instead.
```

**Comment by eheydrick** [comment]
Fixed with https://github.com/sensu-plugins/sensu-plugins-network-checks/pull/35 which will go out in the next release.

**Comment by rs-mrichmond** [comment]
@eheydrick : Got an ETA on that next release?

**Comment by eheydrick** [comment]
Working on cutting a new release.

**Comment by eheydrick** [comment]
Released in 1.0.0.
